.drone.star

```diff
@@ -1046,8 +1047,7 @@ def ocisServices():
 		},
 		'commands': [
 			'cd /var/www/owncloud',
-			'./ocis devldap &',
```

Comment:
should that be deleted too?

Reply:
yes, that should not have been there in the first place.